src/components/collect-qr.vue

```diff
@@ -103,7 +103,7 @@ onMounted(() => {
     ctx.fillText(icon, width * 0.5, width * 0.5, width * 0.2);
 
     // Write text at the bottom
-    ctx.font = '20px Monaco, Menlo, Consolas, "Courier New", monospace';
+    ctx.font = '16px Monaco, Menlo, Consolas, "Courier New", monospace';
```

Comment:
Since the 16 appears in two places, how about we make it a `const`?

Reply:
I wanted to get the PR merged to get it on staging, so I went ahead and made this change. Hope that's OK! Would you mind taking a look at my commit?

Reply:
Looks good!